### PR TITLE
Newsletters: add 271 (2023-10-04)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, update releases/RCs, add topic entries @harding 
- [x] LDK 2069 and 2604 @harding 
- [x] EClair 2756 @murchandamus 
- [x] LDK 2486 @bitschmidty

### CODE REVIEW

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    payment passing through a channel.  If done on many channels, it may
+    be able to track that payment as it crosses the network.  When PSS
+    is used, the attacker would need to track not just the balance of
+    the Alice-and-Bob channel, but also the Alice-and-Carol and
+    Carol-and-Bob channels in order to track the payment.  Even if the
+    attacker did track the balance of all of those channels, the
+    mathematical difficulty of tracking the payment increases, as does
+    the chance that parts of other users' payments that simultaneously
+    pass through those channels could be conflated with parts of the
+    original payment being tracked.  A [paper][pss research] by Van Dam
```
Comment (vostrnad) [inline _posts/en/newsletters/2023-10-04-newsletter.md:57]:
```suggestion
    original payment being tracked.  A [paper][pss research] by van Dam
```

**Comment by vostrnad** [comment]
Should https://github.com/bitcoin/bitcoin/pull/27596 maybe be included as well?

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  would only need to implement the BLIP.  This is similar to Core
+  Lightning's _commando_ plugin (see [Newsletter #210][news210
+  commando]), which allows almost complete remote control of an LN node,
+  but Teinturier envisions his feature as primarily being for control of
+  the most sensitive node actions, such as authorizing a payment---the
+  type of actions where a user would reasonably be willing to go through
+  the hassle of connecting and unlocking a hardware security device and
+  then authorizing the action.  This would make it easier for an end
+  user to secure their LN balance with the same hardware signing device
+  security they used on their onchain balance.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-10-04-newsletter.md:29]:
```suggestion
  security as their onchain balance.
```

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+- **Payment splitting and switching:** Gijs van Dam [posted][van dam pss
+  post] to the Lightning-Dev mailing list about a [plugin][pss plugin] he's
+  written for Core Lightning and some [research][pss research] he's
+  performed related to it.  The plugin allows forwarding nodes to tell
+  their peers that they support _payment splitting and switching_ (PSS).
+  If Alice and Bob share a channel and both of them support PSS, when
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-10-04-newsletter.md:36]:
```suggestion
  If Alice and Bob share a channel and both of them support PSS, then when
```

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    execute _balance discovery attacks_ (BDAs) where a third party
+    repeatedly [probes][topic payment probes] a channel to track its
+    balance.  If done frequently, a BDA can track the amount of a
+    payment passing through a channel.  If done on many channels, it may
+    be able to track that payment as it crosses the network.  When PSS
+    is used, the attacker would need to track not just the balance of
+    the Alice-and-Bob channel, but also the Alice-and-Carol and
+    Carol-and-Bob channels in order to track the payment.  Even if the
+    attacker did track the balance of all of those channels, the
+    mathematical difficulty of tracking the payment increases, as does
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-10-04-newsletter.md:54]:
```suggestion
    computational difficulty of tracking the payment increases, as does
```

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    provide some funds to Alice in the state contract contingent on
+    Alice forwarding the same amount of funds through Bob to Carol in
+    the ordinary LN channel---restoring balance to the LN channel
+    between Alice and Bob.
+
+    One advantage of this approach is that nobody needs to know about the
+    state contract except the participants in each particular contract.
+    To all ordinary LN users, and all forwarding nodes not involved in a
+    particular contract, LN continues to operate using the current
+    protocol.  Another advantage, this time compared to existing channel
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-10-04-newsletter.md:96]:
```suggestion
    protocol.  Another advantage, compared to existing channel
```
Not sure about this suggestion (may be my lack of understanding).

**Review by LarryRuane — COMMENTED** [review]
Just a few of my usual nits... If more content gets pushed, I'll be able to take a look about 12 hours from now, in case that's helpful.

**Comment by murchandamus** [comment]
Pushed to fixup missing words. No additional comments on current draft from me

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.17.0-beta.rc6][] is a release candidate for the next major
+  version of this popular LN node implementation.  A major new
+  experimental feature planned for this release, which could likely
+  benefit from testing, is support for "simple taproot channels".
+
+<!-- FIXME:harding to update Tuesday -->
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-10-04-newsletter.md:135]: FYI, If their blog post is useful:
https://lightning.engineering/posts/2023-10-03-lnd-0.17-launch/

**Comment by harding** [comment]
@vostrnad 

> Should [bitcoin/bitcoin#27596](https://github.com/bitcoin/bitcoin/pull/27596) maybe be included as well?

Thanks for the suggestion!  We'll cover that in next week's newsletter.  Our usual [policy](https://gist.github.com/harding/0dffdde00761262ca094219c33b8ff22) is to only include PRs that were merged when I started writing the current week's newsletter.

File: _posts/en/newsletters/2023-10-04-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+    payment passing through a channel.  If done on many channels, it may
+    be able to track that payment as it crosses the network.  When PSS
+    is used, the attacker would need to track not just the balance of
+    the Alice-and-Bob channel, but also the Alice-and-Carol and
+    Carol-and-Bob channels in order to track the payment.  Even if the
+    attacker did track the balance of all of those channels, the
+    mathematical difficulty of tracking the payment increases, as does
+    the chance that parts of other users' payments that simultaneously
+    pass through those channels could be conflated with parts of the
+    original payment being tracked.  A [paper][pss research] by Van Dam
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2023-10-04-newsletter.md:57]: Yeah.  It seemed to me that we wanted to link the paper in the initial paragraph, but we also wanted to link it here when we first directly reference it.  We've ocassionally double linked like this before, so I think it's ok.

**Comment by harding** [comment]
Pushed edits for all reviews (thanks @vostrnad @bitschmidty @LarryRuane !), added remaining PR descriptions, lede, and topic entries.  Reviewed contributions by @murchandamus and @bitschmidty (thanks!).

Thanks everyone for working on this leet PR!

Thanks everyone

**Comment by bitschmidty** [comment]
Small topic text fixup. Squashed. Merged. Thanks all!

**Pull request merged**